iceoryx2-bb/container/tests/byte_string_tests.rs

```diff
@@ -500,4 +501,14 @@ mod fixed_size_byte_string {
         let mut sut = Sut::from_bytes_truncated(b"Who did eat the last unicorn?");
         sut.remove_range(48, 12);
     }
+
+    #[test]
+    fn placement_default_works() {
+        let mut sut = RawMemory::<Sut>::new_filled(0xff);
+        unsafe { Sut::placement_default(sut.as_mut_ptr()) };
+        assert_that!(unsafe {sut.assume_init()}, len 0);
```

Comment:
```suggestion
        let sut = unsafe { sut.assume_init() };
        assert_that!(sut, len 0);
```
To not have to call assume_init consecutively

Reply:
For now, I would keep it as it is ... so that my future self has something to dislike me for ;)